demisto_sdk/commands/prepare_content/list_unifier.py

```diff
@@ -0,0 +1,51 @@
+import copy
+from pathlib import Path
+from typing import Dict, Optional
+
+from demisto_sdk.commands.common.constants import LISTS_DIR, MarketplaceVersions
+from demisto_sdk.commands.common.files.text_file import TextFile
+from demisto_sdk.commands.common.logger import logger
+from demisto_sdk.commands.prepare_content.unifier import Unifier
+
+
+class ListUnifier(Unifier):
+    @staticmethod
+    def unify(
+        path: Path, data: Dict, marketplace: MarketplaceVersions = None, **kwargs
+    ) -> Dict:
+        logger.info(f"Unifying {path}...")
+        if path.parent.name == LISTS_DIR:
+            return data
+        package_path = path.parent
+        file_content_data_path = ListUnifier.find_file_content_data(
+            package_path / (package_path.name + "_data")
+        )
+        if not file_content_data_path:
+            logger.warning(
+                f"No data file found for '{path}', assuming file is already unified."
+            )
+            return data
+
+        json_unified = copy.deepcopy(data)
+        json_unified = ListUnifier.insert_data_to_json(json_unified, file_content_data_path)
+        logger.debug(f"[green]Created unified json: {path.name}[/green]")
+        return json_unified
+
+    @staticmethod
+    def insert_data_to_json(json_unified: Dict, file_content_data_path: Path):
+        if json_unified.get("data", "") not in ("", "-"):
+            logger.warning(
+                "data section is not empty in "
+                f"{file_content_data_path.with_name(f'{file_content_data_path.parent.name}.json')} file. "
+                "It should be blank or a dash(-)."
+            )
```

Comment:
I implemented this part similarly to handling scripts or integrations in unify

the warning should show the user that we have overridden the section `data` that was written in the list file
but on the other hand, do not issue an error because the user run the unify command